handlers/main.yml

```diff
@@ -7,8 +11,4 @@
 - name: restart nginx
   service: name=nginx state=restarted
   become: yes
-  when: nginx_started is undefined or nginx_started.changed == False
-
-- name: reload nginx
-  service: name=nginx state=reloaded
-  become: yes
+  when: not nginx_first_start.changed
```

Comment:
@morrismukiri did you test this change?

Reply:
Yes. I've used this branch in previous deployments

Reply:
Cool. Seems, most likely than not, that will affect new setups (scenarios where it's the first time setting up NGINX). Let me see if I can test on my side.